rest-server/src/main/java/de/servicehealth/epa4all/server/cetp/CETPEventHandler.java

```diff
@@ -36,6 +38,7 @@ public class CETPEventHandler extends AbstractCETPEventHandler {
 
     private static final Logger log = Logger.getLogger(CETPEventHandler.class.getName());
 
+    private final Event<CashierPayload> cashierPayloadEvent;
```

Comment:
Please do not call it cashier. It is just a websocket client. It can be a normal desktop computer or a tablet.